src/Compilers/Core/Portable/DiagnosticAnalyzer/DiagnosticAnalysisContext.cs

```diff
@@ -240,16 +240,7 @@ public enum GeneratedCodeAnalysisFlags
         /// <summary>
         /// Enable reporting diagnostics on generated code.
         /// </summary>
-        ReportDiagnostics = 0x10,
-
-        /// <summary>
-        /// Default analysis mode for generated code.
-        /// </summary>
-        /// <remarks>
-        /// This mode will always guarantee that analyzer action callbacks are enabled for generated code, i.e. <see cref="Analyze"/> will be set.
-        /// However, the default diagnostic reporting mode is liable to change in future.
-        /// </remarks>
-        Default = Analyze | ReportDiagnostics,
+        ReportDiagnostics = 0x02,
```

Comment:
As an aside, i would like better comments on both of these.  It wasn't at all clear how they were different.  Sri explained hte subtlety the in the meeting but i think that's lost here.